recipes/sphinx-jupyterbook-latex/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "sphinx-jupyterbook-latex" %}
+{% set pip_name = name.replace("-", "_") %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
doesn't need lower

Reply:
Done in https://github.com/conda-forge/staged-recipes/pull/15234/commits/7b9c4049f152539d813b8c4bf547b77459ba6f8b